### DISCUSSION TITLE
500 server error on my insights/pulse page of repo

### DISCUSSION THREAD

**Discussion by Jasonnor** [community/community#22855] [New to GitHub]
<p>Hello there,</p>
<p>When I visit my insights/pulse page of this repo:</p>
<p><a href="https://github.com/Jasonnor/th-music-video-generator/pulse" rel="nofollow noopener">https://github.com/Jasonnor/th-music-video-generator/pulse</a></p>
<p>It keeps showing 500 error, what can I do for it?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/d21cbee52e36561cae8ea916d742d7be9add249d.png" data-download-href="/uploads/short-url/tYJR4ZYzmdXEnrUtlbwVgC52b1P.png?dl=1" title="0.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7609173/181073557-75fa21e9-184c-479c-9d02-801b97f87148.png" alt="0.png" data-base62-sha1="tYJR4ZYzmdXEnrUtlbwVgC52b1P" width="690" height="446" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/d21cbee52e36561cae8ea916d742d7be9add249d_2_690x446.png, https://aws1.discourse-cdn.com/github/optimized/1X/d21cbee52e36561cae8ea916d742d7be9add249d_2_1035x669.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/d21cbee52e36561cae8ea916d742d7be9add249d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/d21cbee52e36561cae8ea916d742d7be9add249d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">0.png</span><span class="informations">1052×681 360 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/jasonnor">@jasonnor</a>,</p>
<p>If you encounter any 500 errors on GitHub, the best thing that you can do is <a href="https://github.com/contact" rel="nofollow noopener">report that problem to GitHub Support</a> and tell them everything you can about the sequence of events that brought you to that page. They’ll be able to tell what’s throwing an error and will likely be able to fix it.</p>
<p>Thanks!</p>
<p>Pat</p>
